CONTRIBUTING.md

```diff
@@ -132,6 +136,7 @@ project's service account.
 - `GCLOUD_TESTS_GOLANG_FIRESTORE_KEY`: The path to the JSON key file of the
 Firestore project's service account.
 - `GCLOUD_TESTS_API_KEY`: API key for using the Translate API created above.
+- `GCLOUD_TESTS_GOLANG_SECONDARY_BIGTABLE_PROJECT_ID`: Developers Console project's ID (e.g. doorway-cliff-677) for Bigtable optional secondary project. This can be same as Firestore project or any project other than the general project.
```

Comment:
Why isn't an env var needed for a Bigtable API key, similar to the other projects?

Reply:
Only Translate API is using API key for running tests.
None of the Bigtable tests use the API key for authentication.

Reply:
Sorry, I meant JSON key file.

Reply:
It looks to me like the Bigtable test might now depend on `GCLOUD_TESTS_GOLANG_FIRESTORE_KEY`, is that correct?